stable_baselines3/common/off_policy_algorithm.py

```diff
@@ -96,6 +95,7 @@ def __init__(
         support_multi_env: bool = False,
         create_eval_env: bool = False,
         monitor_wrapper: bool = True,
+        policy_aliases: Dict[str, Type[BasePolicy]] = {},
```

Comment:
mutable as default is evil in python ;)
https://github.com/satwikkansal/wtfpython#-beware-of-default-mutable-arguments